src/database/models/event_courses.js

```diff
@@ -3,6 +3,9 @@ export default (sequelize, DataTypes) => {
     title: {
       type: DataTypes.STRING,
     },
+    image: {
```

Comment:
why image is text 
is this just a url ?

Reply:
yes its uploaded on cloudinary

Reply:
so why the datatype is text?

Reply:
ok now I see  I wil edit it